lib/engine/game/g_18_carolinas.rb

```diff
@@ -7,6 +7,37 @@ module Engine
   module Game
     class G18Carolinas < Base
       load_from_json(Config::Game::G18Carolinas::JSON)
+
+      def init_companies(players)
+        super.take(players.size)
+      end
+
+      def setup
+        @companies.each_with_index do |company, index|
+          price = company.value
+          player = @players[index]
+          company.owner = player
+          @players[index].companies << company
+          @players[index].spend(price, @bank)
+          @log << "#{player.name} buys #{company.name} for #{format_currency(price)}"
+        end
+
+        @corporations[0..3].sort_by! { @game.rand }
+        @corporations[4..7].sort_by! { @game.rand }
+
+        @corporations.each_with_index do |corporation, index|
+          index -= 4 if index > 3
+          @stock_market.set_par(corporation, @stock_market.par_prices[3 - index])
+
+          @log << "#{corporation.name} is set to par #{@stock_market.par_prices[3 - index].price}"
+          corporation.tokens.pop(1) if index  == 1 || index == 2
+          corporation.tokens.pop(2) if index  == 3
+        end
+
+        @corporations.sort_by! { |corporation| -corporation.par_price.price }
+
+        next_round!
```

Comment:
you shouldn't call this, you should just not have an auction round

Reply:
fixed